fdbserver/workloads/ConfigureDatabase.actor.cpp

```diff
@@ -220,6 +228,8 @@ struct ConfigureDatabaseWorkload : TestWorkload {
 	int additionalDBs;
 	bool allowDescriptorChange;
 	bool allowTestStorageMigration;
+	bool waitStoreTypeCheck;
+	bool downgradeTest1; // if this is true, don't pick up downgrade incompatible config
```

Comment:
Why is there a `1` at the end of the variable name?

Reply:
It means this test runs in the first phase of the downgrade test.